packages/cli/src/commands/generate/scaffold/templates/components/Names.tsx.template

```diff
@@ -27,19 +30,20 @@ const formatEnum = (values: string | string[] | null | undefined) => {
   }
 }
 
-const truncate = (text) => {
-  let output = text
-  if (text && text.length > MAX_STRING_LENGTH) {
+const truncate = (value: string | number) => {
+  let output = value.toString()
```

Comment:
This is blowing up if the DB field is `NULL` since `toString()` doesn't exist on `null`.

Maybe something like this instead?

```suggestion
  let output = value ? value.toString() : ''
```

Reply:
Does this work for you @cannikin ? `  let output = value?.toString() ?? ''`